lib/simulation/openfisca/mapping/foyerFiscal.js

```diff
@@ -11,9 +12,10 @@ module.exports = {
     },
     personnes_a_charge: {
         fn: function(situation) {
-            var enfants = _.pluck(_.filter(situation.individus, { role: 'enfant' }), '_id');
-            var personnesACharge = _.pluck(_.filter(situation.individus, { role: 'personneACharge' }), '_id');
-            return enfants.concat(personnesACharge);
+            var enfants = _.filter(situation.individus, function(individu) {
+                return mapping.isIndividuValid(individu, situation) && individu.role == 'enfant';
```

Comment:
Either remove personnesACharge or make it do what it says. Inconsistency is dangerous.